null_safety_examples/_animation/basic_radial_hero_animation/main.dart

```diff
@@ -0,0 +1,160 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that
+// can be found in the LICENSE file.
+
+// A simple radial transition. The destination route is basic,
+// with no Card, Column, or Text.
+
+import 'dart:math' as math;
+
+import 'package:flutter/material.dart';
+import 'package:flutter/scheduler.dart' show timeDilation;
+
+class Photo extends StatelessWidget {
+  Photo({ Key? key, required this.photo, this.color, this.onTap }) : super(key: key);
+
+  final String photo;
+  final Color? color;
```

Comment:
I'm curious what this colour is used for?

Reply:
Good catch! I think I removed the unused "color" in another example but missed this one